core/include/drivers/wdt.h

```diff
@@ -0,0 +1,24 @@
+/* SPDX-License-Identifier: BSD-2-Clause */
+/*
+ * Copyright 2019 Broadcom.
+ */
+
+#ifndef DRIVERS_WDT_H
+#define DRIVERS_WDT_H
+
+#include <kernel/interrupt.h>
+#include <mm/core_memprot.h>
+
+struct wdt_chip {
+	struct wdt_ops *ops;
```

Comment:
is this `ops` nedded here?

Reply:
I think it's good to have ops under `struct wdt_chip`. `struct wdt_chip` can be re-used by other wdt drivers in future. I have followed core/include/drivers/serial.h.